lib/elixir/lib/map.ex

```diff
@@ -91,6 +91,11 @@ defmodule Map do
       iex> %{map | three: 3}
       ** (KeyError) key :three not found
 
+  The functions in this module that need to find a specific key work in logarithmic time.
+  This means that the time it takes to find keys grows as the map grows, but it's not directly proportional to the map size.
+  In comparision to finding an element in a list, it performs better because lists
```

Comment:
You forgot to format the line above. And:

```suggestion
  In comparison to finding an element in a list, it performs better because lists
```